docs/developers/configuration.md

```diff
@@ -1,14 +1,74 @@
 # Configuration
 
-Configuration is handled by [Viper](https://github.com/spf13/viper), which supports [many ways](https://github.com/spf13/viper#why-viper) of setting config values.
+Prebid Server is built using [Viper](https://github.com/spf13/viper) for configuration. Viper supports JSON, TOML, YAML, HCL, INI, envfile or Java properties formats. YAML, JSON, and Environment Variables are the most popular formats and are used as examples in this guide. 
 
-As a general rule, Prebid Server will log its resolved config values on startup and exit immediately if they're not valid.
+Configuration is logged to standard out as Prebid Server starts up. If a validation error is detected, the application will immediately exit and report the problem.
 
-For development, it's easiest to define your config inside a `pbs.yaml` file in the project root.
+For development, it's easiest to define your config inside a `pbs.yaml` file in the project root. This file is marked to be ignored by `.gitignore` and will not be automatically included in commits.
 
-## Available options
+# We're Working On It
```

Comment:
Nitpick: I was a little confused with this header. Initially I thought it would include stuff we're working on related to configuration, but I think it's actually just making the point that this Configuration guide is still being worked on.

I don't think it's necessary to change this header's title, but just wanted to point it out in case there's a better title (ex. Resources While This Guide is Being Completed)?